docs/image_plugin.md

```
@@ -0,0 +1,48 @@
+pluginを追加する場合は、
+
+- このdirにPluginのソースを追加
+- このdirのindex.tsのimagePluginsに追加
+- src/types/schema.tsのmulmoImageAssetSchemaに追加
+
+
+# Image Pluginの実装
+
+Image Pluginは、
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)